### PR TITLE
Add missing priorities and fail on new ones

### PR DESCRIPTION
**Pull request by pastk**
Needs https://github.com/organicmaps/kothic/pull/22

### CODE REVIEW

File: data/styles/vehicle/include/priorities_3_FG.prio.txt [hunk 1]
```diff
@@ -274,6 +274,8 @@ highway-track-tunnel                                # line z16-
 === 130
 
 highway-construction                                # line z13-
+psurface-unpaved_bad                                # line z14-
+psurface-unpaved_good                               # line z14-
```
Comment (vng) [inline data/styles/vehicle/include/priorities_3_FG.prio.txt:278]: Do we need it?
- These types are not in mapcss styles and not in out drules
- There are 4 psurface subtypes

**Review by vng — COMMENTED** [review]
I merged kothic

File: data/styles/vehicle/include/priorities_3_FG.prio.txt (hunk 1 above)
Comment (vng) [reply data/styles/vehicle/include/priorities_3_FG.prio.txt:278]: UPD: Aha, actually they are in the vehicle style. Could you please show some screenshots, I am just curious how it looks like :)

Comment (vng) [reply data/styles/vehicle/include/priorities_3_FG.prio.txt:278]: I suspect that these line styles should be _higher_ than all highway=* lines.

Comment (vng) [reply data/styles/vehicle/include/priorities_3_FG.prio.txt:278]: So, can we move it higher and see the picture?

Comment (pastk) [reply data/styles/vehicle/include/priorities_3_FG.prio.txt:278]: Yes, but I don't want to do it in this PR :)
Let's keep it simple and about validation only.

**Review by vng — APPROVED** [review]

**Pull request merged**